frontend/src/components/Shared/CrossingMapPage/CrossingMapPage.js

```diff
@@ -173,6 +196,12 @@ class CrossingMapPage extends Component {
                     searchQueryUpdated={this.searchQueryUpdated}
                     selectedCrossingName={selectedCrossingName}
                     setSelectedCommunity={this.setSelectedCommunity}
+                    toggleSearchFocus={() => null}
```

Comment:
What does this do?

Reply:
There's a call to props.toggleSearchFocus which is used in the sidebar component to change state.searchFocused and hide/show the non-autosuggest stuff (nearby crossings/filter checkboxes etc.)

In the mobile view we don't have any sidebar stuff implemented so setting this doesn't do anything, but calling a nonexistent function causes crashes, calling a function that does nothing avoids the crash